src/linagora.esn.unifiedinbox/app/directives/lists.spec.js

```diff
@@ -449,6 +449,7 @@ describe('The linagora.esn.unifiedinbox List module directives', function() {
         $scope.item = {
           moveToMailboxWithRole: sinon.spy(function() {return $q.when();}),
           isUnread: true,
+          mailboxIds: [],
```

Comment:
I don't see any role in this PR. Can you explain it?

Reply:
The `ctrl.canTrashMessages()` call in the `src/linagora.esn.unifiedinbox/views/email/list/list-item.pug` template do calls to an inner function that requires this property on the item object.

Reply:
Thank for the explanation!